### ISSUE TITLE
AuxAddress is not read from network and trigger a re-apply every time

### ISSUE THREAD

**Issue by dubo-dubon-duponey** [kreuzwerker/terraform-provider-docker#10]
Example:

```hcl
-/+ resource "docker_network" "dubo-vlan" {
        attachable      = false
        check_duplicate = true
        driver          = "ipvlan"
      ~ id              = "2e5dda2e5a90ce693456bad29cc3c5afd6c8f9a58568a72325e6569d375cea78" -> (known after apply)
      - ingress         = false -> null
        internal        = false
        ipam_driver     = "default"
        ipv6            = false
        name            = "dubo-lan-vlan"
        options         = {
            "ipvlan_mode" = "l2"
            "parent"      = "wlan0"
        }
      ~ scope           = "local" -> (known after apply)

      + ipam_config { # forces replacement
          + aux_address = {
              + "dns"  = "10.0.4.42"
              + "link" = "10.0.4.43"
            }
          + gateway     = "10.0.4.1"
          + ip_range    = "10.0.4.42/31"
          + subnet      = "10.0.4.1/24"
        }
      - ipam_config { # forces replacement
          - aux_address = {} -> null
          - gateway     = "10.0.4.1" -> null
          - ip_range    = "10.0.4.42/31" -> null
          - subnet      = "10.0.4.1/24" -> null
        }
    }
```

Here is the existing docker network that was created earlier:

```
pi@nightingale:~ $ docker inspect dubo-lan-vlan
[
    {
        "Name": "dubo-lan-vlan",
        "Id": "2e5dda2e5a90ce693456bad29cc3c5afd6c8f9a58568a72325e6569d375cea78",
        "Created": "2020-12-02T03:01:25.262796547Z",
        "Scope": "local",
        "Driver": "ipvlan",
        "EnableIPv6": false,
        "IPAM": {
            "Driver": "default",
            "Options": null,
            "Config": [
                {
                    "Subnet": "10.0.4.1/24",
                    "IPRange": "10.0.4.42/31",
                    "Gateway": "10.0.4.1",
                    "AuxiliaryAddresses": {
                        "dns": "10.0.4.42",
                        "link": "10.0.4.43"
                    }
                }
            ]
        },
        "Internal": false,
        "Attachable": false,
        "Ingress": false,
        "ConfigFrom": {
            "Network": ""
        },
        "ConfigOnly": false,
        "Containers": {},
        "Options": {
            "ipvlan_mode": "l2",
            "parent": "wlan0"
        },
        "Labels": {}
    }
]
```

**Comment by dubo-dubon-duponey** [comment]
This is likely related to #7 and #8

**Comment by dubo-dubon-duponey** [comment]
Closing since the problem came from the fix in #8
